data/WI/incentives.json

```diff
@@ -0,0 +1,667 @@
+[
+  {
+    "id": "WI-1",
```

Comment:
The descriptions on WI-1 through WI-7 are a little short IMO. We usually like to include more data than just the price and item when it's available.

My rule of thumb is that you don't have always try to hit the 150 character limit, but if you're under 50 characters, it should only be because the website didn't give any other information. I do think there are other potentially interesting program requirements in this case.

Reply:
I elaborated on a few of the descriptions, but I couldn't find anything else to say about attic or wall insulation.

Reply:
FYI, there's a lot of extra info that might be relevant in the Program Requirements tab on that website (under Rebate Details).

The hierarchy of what to include is more art than science, but it's nice to have some more detail to offer the user before they need to go hunting themselves.